packages/tocco-ui/src/Select/StyledReactSelect.js

```diff
@@ -21,8 +24,21 @@ const reactSelectStyles = outerTheme => {
   const text = generateInteractionColor(outerTheme.colors.text, {
     action: getLuminance(outerTheme.colors.paper) > 0.5 ? 'darken' : 'lighten'
   })
-
+  const infoText = outerTheme.colors.signal.info.text
   return {
+    control: (base, state) => ({
+      ...base,
+      borderColor: state.isFocused
+        ? infoText
+        : state.theme.colors.neutral20,
+      boxShadow: state.isFocused
```

Comment:
you can add optional attributes with the spread operator like this:
{
 ...(condition? {attributeName : 'value'} : {})
        }

if condition is false, no attribute is added.